### PR TITLE
fix: use image icon

### PR DESCRIPTION
**Pull request by jwyjoy**
I did not experience. Ruby.
But, I would like to use an image icon for my slack.
can this merge?
Do you have a similar plan this?

### CODE REVIEW

File: lib/fluent/plugin/out_slack.rb [hunk 1]
```diff
@@ -15,7 +15,8 @@ class SlackOutput < Fluent::BufferedOutput
     config_param :token,                :string, default: nil # api token
     config_param :username,             :string, default: 'fluentd'
     config_param :color,                :string, default: 'good'
-    config_param :icon_emoji,           :string, default: ':question:'
+    config_param :icon_emoji,           :string, default: nil # if Image use null
```
Comment (sonots) [inline lib/fluent/plugin/out_slack.rb]: this image does not exist, so this should not become default

**Comment by sonots** [comment]
Test codes are required.

**Comment by sonots** [comment]
I understand your demand. I'm willing to merge.

File: lib/fluent/plugin/out_slack.rb (hunk 1 above)
Comment (sonots) [inline lib/fluent/plugin/out_slack.rb]: keep icon_emoji as default, and use icon_url if users configure it.

**Comment by jwyjoy** [comment]
LOL!  Thanks your quick advice :D
I'll try. Test and keep icon_emoji as default.

**Comment by jwyjoy** [comment]
i'll fix some issues that the request again.

**Comment by jwyjoy** [comment]
look at this commit plz.
fix: use default icon emoji. But priority is given to the url

**Comment by sonots** [comment]
I will merge and then write a test for this. thanks!

**Comment by sonots** [comment]
Added tests and documents > https://github.com/sowawa/fluent-plugin-slack/commit/a8b729d217985da5f724d829559f7d3b9fc7b8bf

**Comment by sonots** [comment]
@jwyjoy released v0.5.2!

**Comment by jwyjoy** [comment]
yeah! tnk! @sonots 
:D

**Pull request merged**